tools/digester/main.go

```diff
@@ -274,13 +276,29 @@ func readOneDigest(ctx context.Context, image *Image, index int, wg *sync.WaitGr
 	imgRef, err := docker.ParseReference(fullName)
 	exitOnError(err, "failed to parse container reference")
 
-	digest, err := docker.GetDigest(ctx, nil, imgRef)
+	digest, err := retryGetDigest(5, 1, ctx, nil, imgRef)
+
 	exitOnError(err, "failed to get digest from image")
 
 	ch <- message{index: index, digest: digest.Hex(), fullName: fullName}
 	wg.Done()
 }
 
+func retryGetDigest(attempts int, sleep time.Duration, ctx context.Context, sys *types.SystemContext, imgRef types.ImageReference) (digest digest.Digest, err error) {
+	for i := 0; i < attempts; i++ {
+		if i > 0 {
+			fmt.Println("retrying after error:", err)
+			time.Sleep(sleep)
+			sleep *= 2
+		}
+		digest, err = docker.GetDigest(ctx, sys, imgRef)
+		if err == nil {
+			return digest, nil
+		}
+	}
+	return "", fmt.Errorf("aborting after %d attempts, last error: %s", attempts, err)
```

Comment:
Please change `%s` to `%w`

```suggestion
	return "", fmt.Errorf("aborting after %d attempts, last error: %w", attempts, err)
```

Reply:
done, thanks